js/apps/admin-ui/src/sessions/SessionsTable.tsx

```diff
@@ -185,8 +204,16 @@ export default function SessionsTable({
         }
         columns={columns}
         actionResolver={(rowData: IRowData) => {
-          if (rowData.data.type === "OFFLINE") {
-            return [];
+          if (
+            rowData.data.type === "Offline" ||
+            rowData.data.type === "OFFLINE"
```

Comment:
Why do we have two similar types here? This seems like an issue with the data provided, can it not be normalized in the REST API, or even the database?

Reply:
There is an inconsistency how sessions are obtained on `sessions` page and on `clients/{clientid}/sessions` page. As a result there are different sessions types. Created https://github.com/keycloak/keycloak/issues/28124
For this PR, I had to work with what we have at place ATM as this needs to be ready for 24 backport ASAP.